### PR TITLE
Fix mktempsfile bug

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by itkovian** [comment]
Tests not working?

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
There's no PR tester configured yet for this repo?

**Comment by JensTimmerman** [comment]
ah, we have tests now, jenkins test this please

File: lib/vsc/mympingpong/tools.py [hunk 1]
```diff
@@ -45,8 +45,10 @@ def hwlocmap():
 
     Returns a dict that maps the absolute Processor Unit ID to its socket-id and its core-id
     """
-    xmlout = tempfile.mkstemp(prefix="hwloc-xml-", suffix=".xml")
-
+    # Only need a filename
+    (fh, xmlout) = tempfile.mkstemp(prefix="hwloc-xml-", suffix=".xml")
+    os.close(fh)
```
Comment (JensTimmerman) [inline lib/vsc/mympingpong/tools.py:50]: I would like it more  if we could pass the file handle along, instead of closing it and reopening it

Comment (stdweird) [inline lib/vsc/mympingpong/tools.py:50]: but you can't write to it if it's opened

Comment (JensTimmerman) [inline lib/vsc/mympingpong/tools.py:50]: `os.write(fh, 'foobar')`

**Comment by JensTimmerman** [comment]
jenkins, test this please

**Pull request merged**